MonoGame.Framework/Platform/SDL/SDLGamePlatform.cs

```diff
@@ -49,6 +53,11 @@ public SdlGamePlatform(Game game)
             if (version >= 205 && CurrentPlatform.OS == OS.Windows && Debugger.IsAttached)
                 Sdl.SetHint("SDL_WINDOWS_DISABLE_THREAD_NAMING", "1");
 
+            // https://wiki.libsdl.org/SDL_DropEvent
+            // SDL_DROPTEXT, SDL_DROPBEGIN, and SDL_DROPCOMPLETE are available since SDL 2.0.5. 
+            _multidropSupported = version >= 205;
```

Comment:
You don't need to check this, we already request that SDL version is 2.0.5 or higher few lines above.